### ISSUE TITLE
Validation loss decreasing but training loss is fluctuating a lot.

### ISSUE THREAD

**Issue by kannadaraj** [jik876/hifi-gan#60]
Hi.
 Thanks a ton for sharing this amazing repository. Great work. 
 I am training on a 20 hour data corpus.  Validation loss is going down but training loss is wavering around. Is this normal same value.

![image](https://user-images.githubusercontent.com/17844576/108553076-08f04100-72a7-11eb-9948-010011392998.png)

**Comment by jik876** [comment]
Thank you very much. And sorry for the late reply.
In the early stage of training, losses can increase and decrease depending on the dataset, so it would be nice to watch the loss trend more.
